components/ScheduleHome.vue

```diff
@@ -21,6 +32,29 @@
         </div>
       </div>
     </div>
+    <div v-if="false" class="old-schedule">
+      <div v-for="(day, index) in showsByDate" :key="day.id" class="mb-2 scheduleblock">
+        <div class="flex flex-row justify-between dayname">
+          <h4>{{ dayNames[day[0].day-1] }}</h4>
+          <span>{{ format(new Date(index === 0 ? todayDate :
+            tommorrow.setDate(tommorrow.getDate() + 1)) , 'MMMM dd.') }}</span>
+        </div>
+        <div class="dayshows">
+          <div v-for="show in day" :key="show.id" :class="showAirCheck(index, show.name) ? 'dayblock onair' : 'dayblock'">
+            <div class="show-basic-infos">
+              {{ removeSeconds(show.start) }}-{{ removeSeconds(show.end) }} -
+              <nuxt-link :to="'/shows/' + show.archive_lahmastore_base_url">
+                {{ show.name }}
+              </nuxt-link>
+              <div class="show-image" :style="{ backgroundImage: `url(${show.cover_image_url})` }" />
+            </div>
+            <div v-if="showAirCheck(index, show.name)" class="onairtext">
```

Comment:
Maybe if the show is on air we could bold the text as well? Or display it w/ bigger font? Or would that be too much of an effect? Maybe it is better to be subtle. Well anyway this is definitely not for me to decide

Reply:
question for Csenge, let her work also :-)

Reply:
Agreed and discussed in person!